Searching/BinarySearch.cpp

```diff
@@ -1,7 +1,13 @@
 /*
-    Binary search:
-    A search algorithm that finds the position of a target value
-    within a sorted array
+    <Binary Search>:
+    <A search algorithm that finds the position of a target value
+    within a sorted array>
+
+    <Time complexity>:
+    O(logn), where <n is the number of elements in the array>    
```

Comment:
I suggest writing `O(logn)` as either `O(logN)`, `O(log(n))` or `O(log(N))` for more clarity.

`O(log(N))` would be the best way IMO.